lib/gui/pages/main/styles/_main.scss

```diff
@@ -18,7 +18,16 @@
   margin-top: 75px;
 }
 
-.svg-icon[disabled] path {
+svg {
+  width: 100%;
+  height: 100%;
+}
+
+div.center-block > svg-icon {
+  display: inline-block;
+}
+
+div[disabled] > svg-icon path {
```

Comment:
Hm, what is the common way to solve this problem on React? Is manually implementing a disable property the best practice? Can use some sort of mixin functionality to avoid repeating ourselves?